worlds/tits_the_3rd/items.py

```diff
@@ -72,26 +120,24 @@ def get_item_id(item_name: ItemName):
     "Equipment": set(equipment_table.keys()),
     "Quartz": set(quartz_table.keys()),
     "Currency": set(currency_table.keys()),
+    "Characters": set(character_table.keys()),
+    "Location Unlock": set(location_unlock_table.keys()),
 }
 
+filler_items: List[str] = list(
+    itertools.chain(
+        item_groups["Consumables"],
+        item_groups["Currency"],
+        item_groups["Equipment"],
+        item_groups["Quartz"],
```

Comment:
Hmm, I'm thinking we shouldn't add equipment as filler items (and arguably also not quartz). Receiving a late game equipment on a character can inadvertently trivialize the combat.

Reply:
Equipment is temporary for now since we don't have progressive yet. When we do there won't be any equipment in the pool

Reply:
Quartz I'm sorta fine with you randomly luck sack a late game one early game. It doesn't trivialize the game that much compare to raw stats

Reply:
Let's take equipment out for now still. This type of stuff can pretty easily get forgotten about despite being intended to be temporary.